scripts/travis/08-build-docker-performance-image.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Download stored project artifact on S3 and pack it into Thunder performance docker image
+
+# Get Thunder performance Docker project for packaging
+THUNDER_PERFORMANCE="${TEST_DIR}/../docker-thunder-performance"
+git clone https://github.com/thunder/docker-thunder-performance.git "${THUNDER_PERFORMANCE}"
+
+# Download project artifact from S3
+AWS_ACCESS_KEY_ID="${ARTIFACTS_KEY}" AWS_SECRET_ACCESS_KEY="${ARTIFACTS_SECRET}" aws s3 cp "s3://thunder-builds/${PROJECT_ARTIFACT_FILE_NAME}" "${PROJECT_ARTIFACT_FILE}"
```

Comment:
This is being done in several places now, we should provide the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY globally in the 01-setup-environment.sh file

Reply:
👍